app/views/recipes/index.html.erb

```diff
@@ -1,23 +1,47 @@
-<h1 class="search_results__title">Your search results:</h1>
-<div class="mdl-grid container__recipes">
-  <% @recipes.each do |r| %>
+<% unless @recipes.present? %>
+  <h1 class="search_results__title">Sorry, you have to cook something else...</h1>
+  <h3>Why won't you try:</h3>
+  <div class="mdl-grid container__recipes">
     <div class="mdl-cell mdl-cell--4-col list_item">
       <div class="demo-card-square mdl-card mdl-shadow--2dp">
-        <%= image_tag r.img_url, class: "image" %>
+        <%= image_tag @sample_recipe.img_url, class: "image" %>
           <div class="mdl-card__title mdl-card--expand">
-            <h2 class="mdl-card__title-text"><%= r.name.capitalize %></h2>
+            <h2 class="mdl-card__title-text"><%= @sample_recipe.name.capitalize %></h2>
           </div>
 
-        <div class="mdl-card__supporting-text">
-          <%= truncate(r.description, length: 80) %>
-        </div>
+          <div class="mdl-card__supporting-text">
+            <%= truncate(@sample_recipe.description, length: 80) %>
+          </div>
 
         <div class="mdl-card__actions mdl-card--border">
           <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect">
-            <%= link_to "See full recipe", r, class: "recipe__link" %>
+            <%= link_to "See full recipe", @sample_recipe, class: "recipe__link" %>
           </a>
         </div>
       </div>
-    </div>
+
+  <% else %>
+    <h1 class="search_results__title">Your search results:</h1>
+    <div class="mdl-grid container__recipes">
+    <% @recipes.each do |r| %>
+      <div class="mdl-cell mdl-cell--4-col list_item">
+        <div class="demo-card-square mdl-card mdl-shadow--2dp">
+          <%= image_tag r.img_url, class: "image" %>
+            <div class="mdl-card__title mdl-card--expand">
+              <h2 class="mdl-card__title-text"><%= r.name.capitalize %></h2>
+            </div>
+
+            <div class="mdl-card__supporting-text">
+              <%= truncate(r.description, length: 80) %>
+            </div>
+
+          <div class="mdl-card__actions mdl-card--border">
+            <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect">
+              <%= link_to "See full recipe", r, class: "recipe__link" %>
+            </a>
+          </div>
+        </div>
+      </div>
+    <% end %>
   <% end %>
 </div>
```

Comment:
2 div are unclosed: i think <div class="mdl-cell mdl-cell--4-col list_item"> and <div class="mdl-grid container__recipes">